BeeAppy/__tests__/GetGeoLocation.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import GetGeoLocation from '../src/components/GetGeoLocation';
+
+describe('GetGeoLocation', () => {
+  it('Gets a latitude and logitude coordinate', done => {
+    let getGeoLocation = new GetGeoLocation();
+    // expect.assertions(1);
+    return expect(getGeoLocation.returnLocation()).resolves.toContain("latitude");
+    done();
+
+  })
+})
+// it('Gets a latitude and logitude coordinate', done => {
```

Comment:
These are left in due to the test timing out so I wanted to come back to this and see if it's possible to make the test pass